src/main/resources/openapi/eusage-reports-1.0.yaml

```diff
@@ -54,6 +66,18 @@ paths:
       - $ref: headers/okapi-tenant.yaml
       - $ref: headers/okapi-token.yaml
       - $ref: headers/okapi-url.yaml
+      - in: query
```

Comment:
Do these parameters make sense in the context of POSting to `  /eusage-reports/report-titles/from-counter`?

Reply:
Actually yes.. Since that request for some reason was specified to return the titles !!!

Reply:
OK. Feels weird to me, though — after all, if there are 253 titles but the response only contains 100 (because that's what you asked for) you can hardly redo the POST in order to ask for the next 100.

Reply:
Yep. I think we should return OK or even better NO content if things go smoothly.

Reply:
How about we postpone changing this.. Assuming you are not using the titles in this response (just the fact that it's completed). If this was also an operation happening in the background.. What/who should read this???

Reply:
Fine by me. As you rightly suppose, I do not use the response other than the 200 OK status, and I can't imagine how it could be used. I wonder whether it got into the specs by accident, and no-one ever really had a plan for it?